src/docker-compose/docker-compose-render.ts

```diff
@@ -20,12 +20,13 @@ interface DockerComposeFileServiceSchema {
   readonly dependsOn?: string[];
   readonly build?: DockerComposeBuild;
   readonly image?: string;
-  readonly command?: string[];
+  readonly command?: string | string[];
   readonly volumes?: DockerComposeVolumeMount[];
   readonly networks?: string[];
   readonly ports?: DockerComposeServicePort[];
   readonly environment?: Record<string, string>;
   readonly labels?: Record<string, string>;
+  readonly entrypoint?: string | string[];
```

Comment:
```suggestion
  readonly entrypoint?: string[];
```
We can't have union types due, they don't work in other languages.

Reply:
Ahhh, this makes so much sense now!